apps/backend-e2e/src/user.e2e-spec.ts

```diff
@@ -760,6 +768,14 @@ describe('User', () => {
           token: u1Token
         }));
 
+      it('should 400 if the authenticated user is the target user', async () => {
+        await req.patch({
+          url: `user/follow/${u1.id}`,
+          status: 400,
+          token: u1Token
+        });
+      });
```

Comment:
This test will fail, because PATCH method uses updateFollow, which wasn't modified. Also, this method is not protected at all, it easily throws 500 if relation doesn't exist, we should probably change that too, but I'll need to speak with Tom about it. For now, just remove this test.

Reply:
Actually this test only works because body was not provided, which does return 400 error

Reply:
Good catch, would it be worth asserting on error message in the POST test? I'm not right now because I was just following the format of some of the other test cases but it would ensure we're getting the expected 400 response, not just any 400.